scrapy/utils/test.py

```diff
@@ -76,7 +76,6 @@ class TestSpider(Spider):
 def get_crawler(
     spidercls: Optional[Type[Spider]] = None,
     settings_dict: Optional[Dict[str, Any]] = None,
-    prevent_warnings: bool = True,
```

Comment:
I think we might want to keep this one here, as this is part of the public API. Not only for backward compatibility, but also forward compatibility, maybe we will need to set some default settings like these in the future.

Although you will probably need to add `del prevent_warnings` in the beginning of the implementation so that linters do not complain about the unused parameter.

Reply:
Seems that `pass` works, let's see.